app/services/discovery_report.rb

```diff
@@ -70,26 +66,19 @@ def registration_check(druid)
   # @return [String] primitive version
   def header
     fields = ['Object Container', 'Number of Items', 'Files with no ext', 'Files with 0 Size', 'Total Size', 'Files Readable']
-    fields.concat ['Label', 'Source ID'] if using_manifest?
+    fields.concat ['Label', 'Source ID']
     fields.concat ['Num Files in CM Manifest', 'All CM files found'] if using_smpl_manifest?
-    fields.concat ['Duplicate Filenames?', 'DRUID', 'Registered?', 'APO exists?']
-    fields << 'SourceID unique in DOR?' if using_manifest?
+    fields.concat ['Duplicate Filenames?', 'DRUID', 'Registered?', 'APO exists?', 'SourceID unique in DOR?']
     fields.join(' , ')
   end
 
   # For use by template
   def skipped_files
-    files = ['Thumbs.db', '.DS_Store'] # if these files are in the bundle directory but not in the manifest, they will be ignorned and not reported as missing
+    files = ['Thumbs.db', '.DS_Store', manifest] # if these files are in the bundle directory but not in the manifest, they will be ignorned and not reported as missing
```

Comment:
looks good.  slight preference for manifest first (and I think :manifest might need to get a delegation in bundle.db?  Also the comment reads a little weird now (and has a typo) -- maybe just change it to a method comment?